client/src/AppBundle/Service/Audit/AuditEvents.php

```diff
@@ -11,6 +11,12 @@ final class AuditEvents
     const CLIENT_DISCHARGED_ADMIN_TRIGGER = 'ADMIN_BUTTON';
     const CLIENT_DISCHARGED_CSV_TRIGGER = 'CSV_UPLOAD';
 
+    const EVENT_USER_EMAIL_CHANGED = 'USER_EMAIL_CHANGED';
+    const EVENT_CLIENT_EMAIL_CHANGED = 'CLIENT_EMAIL_CHANGED';
+
+    const TRIGGER_ADMIN_USER_EDIT = 'ADMIN_USER_EDIT';
+    const TRIGGER_DEPUTY_USER_EDIT = 'DEPUTY_USER_EDIT';
```

Comment:
I like that you prefixed with EVENT and TRIGGER - I think it's worth quickly reworking the discharged ones above to use the same format, and update their references in the code.

Reply:
Looks like you've handled that in the other PR 👍

Reply:
Yeah, I was in two minds on if I should mirror the changes but git would probably say there are conflicts.